src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -143,6 +144,9 @@
 
       <SourcesToCopyToTestDir Include="@(ContentWithTargetPath)" />
```

Comment:
I just added a comment to https://github.com/ellismg/buildtools/commit/6a824644b18126db086065175226f3f5dd6b4ed8 about when part of this became disjointed. I think if we had of used _TestCopyLocalByFileNameWithoutDuplicates still there then you wouldn't need to do what you are doing to fix the overrides.